pkgs/development/python-modules/tinytag/default.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "tinytag";
+  version = "1.10.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "tinytag";
+    repo = "tinytag";
+    rev = "refs/tags/${version}";
+    hash = "sha256-Kg67EwDIi/Io7KKnNiqPzQKginrfuE6FAeOCjFgEJkY=";
+  };
+
+  build-system = [ setuptools ];
+
+  preConfigure = ''
+    mkdir -p .github
+    touch .github/{README.md,LICENSE}
+  '';
```

Comment:
Why is this needed?

Reply:
The build fails it thoses files aren't present

Reply:
I can't reproduce such an error.
Could you please check again?

Reply:
Hmm that definitively not that case for `tinytag`, I think i got confused with `library`, which I was working on (tinytag is a dep of library)

Reply:
That package might have an unnecessary and strange build process.